packages/contracts-bedrock/scripts/kontrol.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+# Setup Kontrol 
+cd packages/contracts-bedrock && ./test/kontrol/kontrol/run-kontrol.sh
```

Comment:
This should be able to run in context of any directory, not just in context of the root of the repo

Reply:
https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/scripts/generate-l2-genesis.sh#L10